src/DatumEither.ts

```diff
@@ -414,15 +424,115 @@ export const datumEither: Monad2<URI> & Traversable2<URI> & EitherM1<D.URI> = {
   reduceRight: reduceRightC,
 };
 
+// TODO: After we bump the min bound to >= 2.10, replace this with individual helper fns
+const eitherTDatum = getEitherM(D.Monad)
+
+/**
+ * @since 3.5.0
+ */
+export const Functor: Functor2<URI> = {
+  URI,
+  map: eitherTDatum.map
+}
+
+
+/**
+ * @since 3.5.0
+ */
+export const Bifunctor: Bifunctor2<URI> = {
+  URI,
+  mapLeft: eitherTDatum.mapLeft,
+  bimap: eitherTDatum.bimap
+}
+
+/**
+ * @since 3.5.0
+ * 
+ * Note: This instance agrees with the standalone Applicative/Chain/Monad instances but _disagrees_ with the deprecated `datum` mega-instance.
+ */
+export const Apply: Apply2<URI> = {
+  ...Functor,
+  ap: eitherTDatum.ap
+}
+
+/**
+ * @since 3.5.0
+ */
+export const Chain: Chain2<URI> = {
+  ...Apply,
+  chain: eitherTDatum.chain
+}
+
+/**
+ * @since 3.5.0
+ */
+export const Applicative: Applicative2<URI> = {
+  ...Apply,
+  of: eitherTDatum.of
+}
+
+/**
+ * @since 3.5.0
+ */
+export const Alt: Alt2<URI> = {
+  ...Functor,
+  alt: eitherTDatum.alt
+}
+
+/**
+ * @since 3.5.0
+ */
+export const Alternative: Alternative2<URI> = {
+  ...Alt,
+  ...Applicative,
+  zero: constInitial
+}
+
+/**
+ * @since 3.5.0
+ */
+export const Monad: Monad2<URI> = {
+  ...Chain,
+  of: eitherTDatum.of
+}
+
+/**
+ * @since 3.5.0
+ */
+export const MonadThrow: MonadThrow2<URI> = {
+  ...Monad,
+  throwError: failure
+}
+
+/**
+ * @since 3.5.0
+ */
+export const Foldable: Foldable2<URI> = {
+  URI,
+  reduce: reduceC,
+  reduceRight: reduceRightC,
+  foldMap: foldMapC
+}
+
+/**
+ * @since 3.5.0
+ */
+export const Traversable: Traversable2<URI> = {
+  ...Functor,
+  ...Foldable,
+  sequence: sequenceC,
+  traverse: traverseC
+}
+
 /**
  * @since 3.2.0
  */
-export const sequenceTuple = sequenceT(datumEither);
+export const sequenceTuple = sequenceT(Apply);
```

Comment:
Looking through this PR again, this is breaking actually. Going to undo this and the below change and push this off until 4.0